roundedimageview/src/main/java/com/makeramen/roundedimageview/RoundedImageView.java

```diff
@@ -74,7 +74,7 @@
   private boolean mMutateBackground = false;
   private int mResource;
   private int mBackgroundResource;
-  private ScaleType mScaleType = ScaleType.FIT_CENTER;
```

Comment:
If set init value for mScaleType and no `android:scaleType` set in xml, `ImageView#init()` would not call `setScaleType()` and in RoundedImageView constructor `setScaleType` would be called but `super.setScaleType()` cannot be called.   
![image](https://cloud.githubusercontent.com/assets/5254121/17205828/5046c6fe-54df-11e6-833e-7e88c987747f.png)

And that leads to the issue bug.